core/modules/widgets/subclasses/edit-text-subclass.js

```diff
@@ -0,0 +1,65 @@
+/*\
+title: $:/core/modules/widgets/subclasses/edit-text-subclass.js
+type: application/javascript
+module-type: widget-subclass
+
+A subclass that intercepts the refresh method of edit-text widgets for enhanced refreshing
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+exports.baseClass = "edit-text";
```

Comment:
It looks like you're extending the existing edit-text widget, rather than creating a new widget that is a subclass of it. In that case, why not just modify the existing edit-text refresh method, rather than creating a subclass? Or is this a residue from a plugin implementation?

Reply:
Yes, it's a residue from a plugin implementation. I kept it this way because I didn't want to make the refresh method too big